pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -65,11 +66,16 @@ func (r *ControlPlaneReconciler) Reconcile() (reconcile.Result, error) {
 		if namespace.Labels == nil {
 			namespace.Labels = map[string]string{}
 		}
+
+		// Label the control plane namespace so that Kiali knows about it
+		namespace.Labels[common.MemberOfKey] = r.Instance.Namespace
+
```

Comment:
We should probably remove these labels during uninstall/delete.

Reply:
Done